Cargo.toml

```diff
@@ -14,7 +14,7 @@ name = "clif-util"
 path = "src/clif-util.rs"
 
 [dependencies]
-cfg-if = "0.1"
+cfg-if = "0.1.6"
```

Comment:
This appears to have been underspecified. With v0.1.4 locally, I was getting errors about missing macros. Presumably due to lack of this commit: https://github.com/alexcrichton/cfg-if/commit/67e01e5f0ea0e27d0bba38d3da7d73cda52d8be4